### PR TITLE
Add additional pip modules to One Time Setup Instructions

### PR DESCRIPTION
**Pull request by craig-br**
##### SUMMARY

Added additional pip modules necessary for Windows workshop when deploying using the AWS provisioner

##### ISSUE TYPE
- Docs Pull Request

##### COMPONENT NAME
- docs

##### ADDITIONAL INFORMATION
ansible-playbook provision_lab.yml -e @../../win_extra_vars/win_extra_vars.yml

```
TASK [workshop_check_setup : make sure (windows) packages required are installed on this control node] *****************************************
changed: [localhost]

TASK [workshop_check_setup : fail if previous check has changed] *******************************************************************************
fatal: [localhost]: FAILED! => changed=false 
  assertion: package_state.changed != True
  evaluated_to: false
  msg: Assertion failed

PLAY RECAP *************************************************************************************************************************************
localhost                  : ok=15   changed=2    unreachable=0    failed=1    skipped=6    rescued=0    ignored=0   
```

### CODE REVIEW

File: provisioner/ansible.cfg [hunk 1]
```diff
@@ -10,7 +10,8 @@
 [defaults]
 interpreter_python = auto_silent
 stdout_callback = yaml
-inventory      = hosts
+inventory      = ./craigbsec/instructor_inventory.txt
```
Comment (IPvSean) [inline provisioner/ansible.cfg:13]: you are going to want to gitignore this file so you don't change the public one @craig-br

File: provisioner/roles/manage_ec2_instances/defaults/main.yml [hunk 2]
```diff
@@ -0,0 +1,196 @@
+---
```
Comment (IPvSean) [inline provisioner/roles/manage_ec2_instances/defaults/main.yml:1]: why did you add this without removing the other one? in `workshops/provisioner/roles/manage_ec2_instances/defaults/main/main.yml`

Comment (craig-br) [reply provisioner/roles/manage_ec2_instances/defaults/main.yml:1]: Apologies. Mistake on my side with commit. I'll fix.
